packages/graphiql-react/src/editor/hooks.ts

```diff
@@ -208,12 +208,22 @@ export function usePrettifyEditors({
     }
 
     if (queryEditor) {
-      const editorContent = queryEditor.getValue();
-      const prettifiedEditorContent = print(parse(editorContent));
-
-      if (prettifiedEditorContent !== editorContent) {
-        queryEditor.setValue(prettifiedEditorContent);
-      }
+      import('prettier/standalone').then(({ default: prettier }) => {
+        import('prettier/parser-graphql').then(({ default: graphqlPlugin }) => {
```

Comment:
I guess you can use `Promise.all` in this case, and async/await.
```suggestion
const [{ default: prettier }, { default: graphqlPlugin }] = await Promise.all([
  import('prettier/standalone'),
  import('prettier/parser-graphql'),
]);
```

Reply:
I would need to make the parent function async as well, I can try this